observer/probers/http/http.go

```diff
@@ -3,19 +3,22 @@ package probers
 import (
 	"fmt"
 	"net/http"
+	"strings"
 	"time"
 )
 
 // HTTPProbe is the exported 'Prober' object for monitors configured to
 // perform HTTP requests.
 type HTTPProbe struct {
-	url    string
-	rcodes []int
+	url       string
+	rcodes    []int
+	useragent string
 }
 
 // Name returns a string that uniquely identifies the monitor.
 func (p HTTPProbe) Name() string {
-	return fmt.Sprintf("%s-%d", p.url, p.rcodes)
+	// Trim trailing hyphen if useragent is not provided
+	return strings.TrimSuffix(fmt.Sprintf("%s-%d-%s", p.url, p.rcodes, p.useragent), "-")
```

Comment:
Now that there's a _default_ user agent, I think you can remove the `strings.Trim()` here

Reply:
Thanks, done!